identity-service/src/recovery.html

```diff
@@ -0,0 +1,670 @@
+<!DOCTYPE html
```

Comment:
eventually we should import these from

https://github.com/AudiusProject/emails

but let's do that later

Reply:
sg to me